src/Files.App/Actions/Content/Selection/SelectAllAction.cs

```diff
@@ -1,6 +1,9 @@
 ﻿// Copyright (c) 2023 Files Community
 // Licensed under the MIT License. See the LICENSE.
 
+using Files.App.ViewModels.UserControls;
+using System.Diagnostics.Eventing.Reader;
```

Comment:
I don't think `this using` is necessary.

Reply:
You are correct. Thanks.